### DISCUSSION TITLE
[Support]:  Docker pull cache-amd64: manifest unknown error

### DISCUSSION THREAD

**Discussion by virtualbubble** [blakeblackshear/frigate#10786] [General Support]
### Describe the problem you are having

Cannot pull the latest image ghcr.io/blakeblackshear/frigate:cache-amd64 I get this error:

manifest unknown

### Version

latest

### Frigate config file

```yaml
N/A
```

### Relevant log output

```shell
N/A
```

### FFprobe output from your camera

```shell
N/A
```

### Frigate stats

```json
N/A
```

### Operating system

Other Linux

### Install method

HassOS Addon

### Coral version

CPU (no coral)

### Network connection

Wired

### Camera make and model

N/A

### Any other information that may be helpful

N/A

**Comment by blakeblackshear** [comment]
That's not the latest image. It's just a random build cache image not meant to be used directly.

You should be using the images listed in the docs: https://docs.frigate.video/frigate/installation#docker
